### DISCUSSION TITLE
How to change the width of a chapter

### DISCUSSION THREAD

**Discussion by carboML** [gohugo-ananke/ananke#570] [2 - Q&A / Support]
Hi!
I am wondering how to change the width of the article or chapter. The images that I have to use would be better looking if the width of the whole article increases. Anyone knows how to do this?

![screenshot](https://user-images.githubusercontent.com/111242250/186472587-7a9e71c0-26c8-486a-8767-34bfffd4eb02.PNG)

Thanks!!

**Comment by hawksheng** [comment]
I just modified file "themes/ananke/layouts/_default/single.html" at line 50.
Replaced the width value w-two-thirds-l of this dev to w-100, and it works. :)
